qt/scientific_interfaces/Indirect/IndirectFitPropertyBrowser.cpp

```diff
@@ -327,9 +327,22 @@ QString IndirectFitPropertyBrowser::selectedFitType() const {
 /**
  * Sets whether fit members should be convolved with the resolution after a fit.
  *
- * @param convolveMembers If true, members are to be convolved.
+ * @param state: If true, members are to be convolved.
```

Comment:
Instead of `state` could you use a parameter name like `convolveEnabled` to be clearer on its use?